extensions/frontend-tiles/src/integration-test/utils/webpack-config.js

```diff
@@ -0,0 +1,73 @@
+// /*---------------------------------------------------------------------------------------------
+//  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+//  * See LICENSE.md in the project root for license terms and full copyright
+//  *notice.
+//  *--------------------------------------------------------------------------------------------*/
+
+const path = require('path');
+const {globSync} = require('glob');
+const frontendLib = path.resolve(__dirname, '../../../lib/cjs');
+const Dotenv = require('dotenv-webpack');
+const copyExternalsPlugin =
```

Comment:
what do you need this plugin for?

Reply:
Wasn't needed, accidental hold out from earlier iterations, it's been removed.